src/Game/UI/Gumps/OptionsGump.cs

```diff
@@ -556,9 +547,88 @@ private void BuildVideo()
 
+            ScrollAreaItem profileItem = new ScrollAreaItem();
+            text = new Label("Performance Profile", true, HUE_FONT)
+            {
+                Y = _showCorpseNameIncoming.Bounds.Bottom + 5 + SPACE_Y
+            };
+            var perfProfile = new Combobox(text.X + text.Width + 10, text.Y, 200, new[] { "High", "Medium", "Low" });
+
+            perfProfile.OnOptionSelected += (sender, option) => {
+                // Common options
+                _reduceFPSWhenInactive.IsChecked = true;
+                _windowBorderless.IsChecked = false;
+                _gameWindowFullsize.IsChecked = false;
+                _sliderZoom.Value = 5;
+                _altLights.IsChecked = false;
+                _darkNights.IsChecked = false;
+                _runMouseInSeparateThread.IsChecked = true;
+
+                switch (option)
+                {
+                    case 0: // High
+                        _sliderFPS.Value = Constants.MAX_FPS;
+                        _zoomCheckbox.IsChecked = true;
+                        _useColoredLights.IsChecked = true;
+                        _enableDeathScreen.IsChecked = true;
+                        _enableBlackWhiteEffect.IsChecked = true;
+                        _enableShadows.IsChecked = true;
+                        _auraType.SelectedIndex = 3;
+                        _auraMouse.IsChecked = true;
+                        _xBR.IsChecked = true;
+                        _hideChatGradient.IsChecked = false;
+                        break;
+                    case 1: // Medium
+                        _sliderFPS.Value = 25;
+                        _zoomCheckbox.IsChecked = false;
+                        _useColoredLights.IsChecked = true;
+                        _enableDeathScreen.IsChecked = true;
+                        _enableBlackWhiteEffect.IsChecked = true;
+                        _enableShadows.IsChecked = false;
+                        _auraType.SelectedIndex = 1;
+                        _auraMouse.IsChecked = false;
+                        _xBR.IsChecked = false;
+                        _hideChatGradient.IsChecked = false;
+                        break;
+                    case 2: // Low
+                        _sliderFPS.Value = Constants.MIN_FPS;
+                        _gameWindowWidth.SetText("800");
+                        _gameWindowHeight.SetText("600");
+                        _zoomCheckbox.IsChecked = false;
+                        _useColoredLights.IsChecked = false;
+                        _enableDeathScreen.IsChecked = false;
+                        _enableBlackWhiteEffect.IsChecked = false;
+                        _enableShadows.IsChecked = false;
+                        _auraType.SelectedIndex = 0;
+                        _auraMouse.IsChecked = false;
+                        _xBR.IsChecked = false;
+                        _hideChatGradient.IsChecked = true;
+                        break;
+                }
+            };
```

Comment:
Can you pull that out into a private method?

Reply:
can case 0 be set to.custom?